spec/lib/submission_builder/ty2024/states/nc/documents/d400_spec.rb

```diff
@@ -65,6 +65,35 @@
         expect(xml.document.at('TotalAmountDue')&.text).to eq "0" # 27
         expect(xml.document.at('Overpayment')&.text).to eq "2500" # 28
         expect(xml.document.at('RefundAmt')&.text).to eq "2500" # 34
+        expect(xml.document.at('PaymentContact PersonName FirstName')&.text).to be_nil
+        expect(xml.document.at('PaymentContact PersonName MiddleInitial')&.text).to be_nil
+        expect(xml.document.at('PaymentContact PersonName LastName')&.text).to be_nil
+        expect(xml.document.at('PaymentContact PersonName NameSuffix')).to be_nil
+        expect(xml.document.at('PaymentContact USPhoneNumber')&.text).to be_nil
+      end
+
+      context "when owed" do
+        let(:intake) { create(:state_file_nc_intake, :taxes_owed) }
+        before do
+          intake.phone_number = nil
```

Comment:
maybe I should set a intake.direct_file_data.phone_number = "9845559876" to be more explicit?

Reply:
✅